### ISSUE TITLE
Can Oliver work out her age?

### ISSUE THREAD

**Issue by elolelo** [olivia-ai/olivia#50]
![Screenshot from 2019-06-27 21-57-43](https://user-images.githubusercontent.com/32453496/60297646-f6cecb80-9928-11e9-993a-5741a3e4c2ff.png)

**Comment by ananagame** [comment]
No, but that's a good module to add! :)

**Comment by phillipahereza** [comment]
@ananagame is this issue up for grabs? If yes, can I work on it?

**Comment by elolelo** [comment]
> @ananagame is this issue up for grabs? If yes, can I work on it?

I've been snowed under, although I was working on it - I am done, so you can go ahead and work on it.

**Comment by ananagame** [comment]
yes you can! just take a look at the doc about Modules and how the other modules are done to respect the format ;)
